### PR TITLE
Fixing some minor errors from previous merge

### PR DESCRIPTION
**Pull request by Navidur1**
Also adjusted pinecone DB functions

### CODE REVIEW

File: requirements.txt
Comment (Ismail-Ben) [inline requirements.txt:1]: why is this changed?

Comment (Navidur1) [reply requirements.txt:1]: U had some extra stuff (numpy)

**Review by adrianchow-tech — APPROVED** [review]
